src/server/page.tsx

```diff
@@ -1,27 +1,30 @@
 // ----- Imports ----- //
 
-import React, { ReactElement, createElement as h } from 'react';
-import { renderToString } from 'react-dom/server';
-import { cache } from 'emotion';
-import { CacheProvider } from '@emotion/core';
-import { extractCritical } from 'emotion-server';
-import { EmotionCritical } from 'create-emotion-server';
-import { JSDOM } from 'jsdom';
-import { Format, Design, Display } from '@guardian/types/Format';
-import { RenderingRequest } from '@guardian/apps-rendering-api-models/renderingRequest';
-import { background } from '@guardian/src-foundations/palette';
-
-import { getThirdPartyEmbeds, requiresInlineStyles, ThirdPartyEmbeds } from 'capi';
-import { fromCapi, Item } from 'item';
-import { Option, some, none, map } from '@guardian/types/option';
-import { compose } from 'lib';
-import Body from 'components/body';
-import Scripts from 'components/scripts';
-import { atomCss, atomScript } from 'components/atoms/interactiveAtom';
-import { csp } from 'server/csp';
-import { pageFonts } from 'styles';
-import Meta from 'components/meta';
-
+import { CacheProvider } from "@emotion/core";
+import type { RenderingRequest } from "@guardian/apps-rendering-api-models/renderingRequest";
+import { background } from "@guardian/src-foundations/palette";
+import { Design, Display } from "@guardian/types/Format";
+import type { Format } from "@guardian/types/Format";
+import type { Option } from "@guardian/types/option";
```

Comment:
This is importing from the same place ("@guardian/types/Format") however broken down separately into new lines. Is this just for the linter or for efficiency?

Reply:
I'm not sure if there's a better way to handle this, but the following wouldn't work because not all of the imports are types

``` import type { Design, Display, Format } from "@guardian/types/Format"; ```

Reply:
yeah this is the rule that requires type-only imports to be explicit